certbot/tests/util.py

```diff
@@ -254,13 +254,16 @@ class ConfigTestCase(TempDirTestCase):
     def setUp(self):
         super(ConfigTestCase, self).setUp()
         self.config = configuration.NamespaceConfig(
-            mock.MagicMock(
-                config_dir=os.path.join(self.tempdir, 'config'),
-                work_dir=os.path.join(self.tempdir, 'work'),
-                logs_dir=os.path.join(self.tempdir, 'logs'),
-                server="example.com",
-            )
+            mock.MagicMock(**constants.CLI_DEFAULTS)
         )
+        self.config.verb = "certonly"
+        self.config.config_dir = os.path.join(self.tempdir, 'config')
+        self.config.work_dir = os.path.join(self.tempdir, 'work')
+        self.config.logs_dir = os.path.join(self.tempdir, 'logs')
+        self.config.cert_path = constants.CLI_DEFAULTS['auth_cert_path']
+        self.config.fullchain_path = constants.CLI_DEFAULTS['auth_chain_path']
+        self.config.chain_path = constants.CLI_DEFAULTS['auth_chain_path']
+        self.config.server = "example.com"
```

Comment:
This was from your other PR, but after thinking about it, why are we setting this value rather than just leaving it at the normal server default?

Reply:
The default points to the letsencrypt ACME server and I usually wouldn't give tests any way to accidentally throw garbage data at a production service. What are your thoughts?

Reply:
My thought was that we should do a pretty good job of mocking out the calls that send data to the production server and it's nice to use the real default when possible so any tests that may rely on that value now or in the future automatically work properly, however, you have a good point. It's unlikely that the value used here will matter in a test and it's an extra safe guard against unnecessary server traffic so you can leave this as is.

Thanks for pointing that out!